### PR TITLE
PMM-6956 Add IA toggle

### PR DESCRIPTION
**Pull request by tiagomotasantos**
[PMM-6956](https://jira.percona.com/browse/PMM-6956)

https://jira.percona.com/browse/PMM-6956

### CODE REVIEW

**Review by atymchuk — APPROVED** [review]

**Review by lunaticusgreen — DISMISSED** [review]
👍

File: pmm-app/src/pmm-settings/components/Advanced/Advanced.test.tsx [hunk 1]
```diff
@@ -47,6 +47,20 @@ describe('Advanced::', () => {
     expect(sttSwitch.prop('disabled')).toBeTruthy();
   });
 
+  it('Cant change alerting when telemetry is off', () => {
```
Comment (nicolalamacchia) [inline pmm-app/src/pmm-settings/components/Advanced/Advanced.test.tsx:50]: typo here :)

Comment (tiagomotasantos) [reply pmm-app/src/pmm-settings/components/Advanced/Advanced.test.tsx:50]: I don't see it @nicolalamacchia

Comment (nicolalamacchia) [reply pmm-app/src/pmm-settings/components/Advanced/Advanced.test.tsx:50]: `Cant` -> `Can't` : )

**Review by nicolalamacchia — APPROVED** [review]

**Review by nicolalamacchia — APPROVED** [review]

**Review by lunaticusgreen — APPROVED** [review]

**Pull request merged**